### ISSUE TITLE
AssertionError module.endswith

### ISSUE THREAD

**Issue by MarcRoigCampos** [mbarkhau/pretty-traceback#1]
Executing python file as `python script.py` works fine. But if I try to do `python ./script.py` it gives the following error

```
Error in sys.excepthook:
Traceback (most recent call last):
  File "C:\Users\M1\anaconda3\envs\myenv\lib\site-packages\pretty_traceback\hook.py", line 23, in excepthook
    tb_str = formatting.exc_to_traceback_str(exc_value, traceback, color)
  File "C:\Users\M1\anaconda3\envs\myenv\lib\site-packages\pretty_traceback\formatting.py", line 281, in exc_to_traceback_str
    return format_tracebacks(tracebacks, color)
  File "C:\Users\M1\anaconda3\envs\myenv\lib\site-packages\pretty_traceback\formatting.py", line 255, in format_tracebacks
    traceback_str = format_traceback(tb_tup, color)
  File "C:\Users\M1\anaconda3\envs\myenv\lib\site-packages\pretty_traceback\formatting.py", line 244, in format_traceback
    ctx = _init_entries_context(traceback.entries)
  File "C:\Users\M1\anaconda3\envs\myenv\lib\site-packages\pretty_traceback\formatting.py", line 156, in _init_entries_context
    rows = list(_iter_entry_rows(aliases, entry_paths, entries))
  File "C:\Users\M1\anaconda3\envs\myenv\lib\site-packages\pretty_traceback\formatting.py", line 135, in _iter_entry_rows
    assert module.endswith(_module)
AssertionError
```

**Comment by mbarkhau** [comment]
Will check, thanks for the report.

**Comment by mbarkhau** [comment]
Fixes with 2020.1011
